src/main/resources/features/theme 1/25 - site review.feature

```diff
@@ -0,0 +1,60 @@
+@regression
+  Feature: 25 - site review
+
+    Scenario: VE user creates a satisfactory site review with an activity performed
+      #Search for a site with an existing site review
+      Given I load "SITE_REVIEW" as {siteNumber}, {siteName}
+      And I login without 2FA using "VEHICLE_EXAMINER_USER" as {vehicleExaminer}
+      And I search for Site information by site number with {siteNumber}
+      #Start a site review
```

Comment:
It would be nice to split these steps up into what is doing on each page, making it easy to see what steps are carried out on what pages